<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Condition</class>
 <widget class="QDialog" name="Condition">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Condition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DeClutter.qrc">
    <normaloff>:/images/icons/DeClutter.ico</normaloff>:/images/icons/DeClutter.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select files by</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="conditionCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>size</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tags</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>type</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nameCombo">
         <item>
          <property name="text">
           <string>matches</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>doesn't match</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="expressionLabel">
         <property name="text">
          <string>expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filemask">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="filenameHint">
       <property name="text">
        <string>You can use multiple comma-separated expressions</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ageLayout">
       <item>
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>File age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ageCombo">
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="age"/>
       </item>
       <item>
        <widget class="QComboBox" name="ageUnitsCombo">
         <item>
          <property name="text">
           <string>days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>months</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>years</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="ageSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sizeLayout">
       <item>
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>File size is</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sizeCombo">
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="size"/>
       </item>
       <item>
        <widget class="QComboBox" name="sizeUnitsCombo">
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TB</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="tagLabel">
         <property name="text">
          <string>File has</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tagsCombo">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>all</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>no tags</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>any tags</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tags in group</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tagLabel2">
         <property name="text">
          <string>of selected tags:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tagGroupsCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="TagTree" name="tagsView">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="selectedTagsLabel">
       <property name="text">
        <string>Selected tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>File type </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeSwitchCombo">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
         </property>
         <property name="minimumContentsLength">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>is</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is not</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagTree</class>
   <extends>QTreeView</extends>
   <header location="global">..tag_tree</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DeClutter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Condition</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Condition</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
